ipatests/test_integration/test_commands.py

```diff
@@ -1286,17 +1286,16 @@ def test_ipa_nis_manage_enable_incorrect_password(self):
         This testcase checks if ipa-nis-manage enable
         command throws error on console for invalid DS admin password
         """
-        msg = (
-            "Insufficient access: Invalid credentials "
-            "Invalid credentials\n"
-        )
+        msg1 = "Insufficient access: "
```

Comment:
@flo-renaud ,
Cannot we have just one string to check this? why we need two?

Reply:
@kaleemsiddiqu 
The error message when using the @389ds/389-ds-base-nightly is: 
`Insufficient access:  Invalid credentials`
 (with 2 spaces between : and I) instead of
`Insufficient access: Invalid credentials Invalid credentials`
I chose to check for the 2 msgs instead of writing a regexp compatible with one space or two spaces after the colon.

Reply:
ok.